policyengine_us/parameters/gov/states/al/tax/income/adjustments.yaml

```diff
@@ -0,0 +1,20 @@
+description: Alabama provides these adjustments for the state adjusted gross income.
```

Comment:
This is not clear. Adjustments could be positive or negative. The law defines them as deductions.